### PR TITLE
feat: implement nft minting

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: src/components/Superfest/NFTClaimingBox/NFTClaimingBox.tsx [hunk 1]
```diff
@@ -83,6 +83,38 @@ export const NFTClaimingBox = ({
...
       </NFTDisplayBox>
+      <Box marginTop={'64px'}>
+        <NFTClaimingHeader>
+          <NFTClaimingTitle>
+            {String('unlock the mystery box').toUpperCase()}
+          </NFTClaimingTitle>
+          <Box marginTop="32px" marginBottom="32px">
+            <NFTClaimingDescription>
+              {
+                'When you mint all Superchain wristbands, you become eligible to mint a unique Superchain Mystery Box.'
```
Comment (admsk2) [inline src/components/Superfest/NFTClaimingBox/NFTClaimingBox.tsx:94]: it does not have to be in brackets i believe

File: src/hooks/useCheckFestNFTAvailability.ts [hunk 2]
```diff
@@ -61,46 +135,92 @@ export const useCheckFestNFTAvailability = ({
...
+      claimInfo[chainName].verifyIds =
+        data.prepareParticipate.mintFuncInfo.verifyIDs?.[0];
+      claimInfo[chainName].powahs =
+        data.prepareParticipate.mintFuncInfo.powahs?.[0];
+      claimInfo[chainName].signature = data.prepareParticipate.signature;
+      claimInfo[chainName].claimingAddress =
+        data.prepareParticipate.spaceStation;
+      claimInfo[chainName].NFTAddress =
+        data.prepareParticipate.mintFuncInfo.nftCoreAddress;
+    }
```
Comment (admsk2) [inline src/hooks/useCheckFestNFTAvailability.ts:220]: That should work too and a bit cleaner:
```
if (data?.prepareParticipate?.allow) {
  const { mintFuncInfo, signature, spaceStation } = data.prepareParticipate;
  const { verifyIDs, powahs, nftCoreAddress } = mintFuncInfo;

  claimInfo[chainName] = {
    isClaimable: true,
    verifyIds: verifyIDs?.[0],
    powahs: powahs?.[0],
    signature,
    claimingAddress: spaceStation,
    NFTAddress: nftCoreAddress,
  };
}
```

**Comment by admsk2** [comment]
Let's also remove console.logs and commented code if not used anymore

**Review by oktapodia — APPROVED** [review]
LGTM

File: src/hooks/useCheckNFTAvailability.ts [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+interface GalxeGraphqlCheckRes {
+  campaign: {
+    numberID: number;
+    participationStatus: string;
+  };
+}
+
+const GALXE_ENDPOINT = 'https://graphigo.prd.galaxy.eco/query';
```
Comment (admsk2) [inline src/hooks/useCheckNFTAvailability.ts:60]: wdyt about putting this into `urls.ts`?

**Review by admsk2 — APPROVED** [review]

**Pull request merged**